kogito-editors-js/yarn.lock

```diff
@@ -4365,6 +4365,11 @@ balanced-match@^1.0.0:
   resolved "https://registry.yarnpkg.com/balanced-match/-/balanced-match-1.0.2.tgz#e83e3a7e3f300b34cb9d87f615fa0cbf357690ee"
   integrity sha512-3oSeUO0TMV67hN1AmbXsK4yaqU7tjiHlbxRDZOpH0KW9+CeX4bRAaX0Anxt0tx2MrpRpWwQaPwIlISEJhYU5Pw==
 
+base16@^1.0.0:
```

Comment:
@vpellegrino are all these new dependencies related to `react-json-view` you added?

Reply:
yes, because of `"react-json-view": "^1.21.3"` used in the showcase application.